### DISCUSSION TITLE
Maybe you can make a perfect supported list to help people buy monitors?

### DISCUSSION THREAD

**Discussion by jerry1230** [MonitorControl/MonitorControl#984] [Ideas]
First of all, thank u for the MonitorControl , Its really awesome.

As apple not willing to add an official software to support volume adjust over digital video output(hdmi or DP) 
If someone want to buy a perfect 3rd party monitor for mac, there is nowhere to get a guidance about that.
So, I think, maybe you can make a perfect supported products list to help people buy monitors...

**Comment by waydabber** [comment]
It's a great idea - it should be a community driven thing though. If you want to spearhead an effort like this (maybe a webform of some kind could be created), I'll gladly input info about my displays. If the devs @the0neyouseek @JoniVR agree, we could promote this on the main page.

**Reply by the0neyouseek** [reply]
I think it's a great idea, @alin23 already has an extensive database of displays accessible here https://db.lunar.fyi/ that could be used as a starting point

**Reply by alin23** [reply]
I've already helped a few people make their purchase decision using the Lunar database. For example you can see if a specific monitor supports DDC by looking at the DisplayPort connections for that monitor.

Here I'm checking the **HP U28 4K** compatibility on **M1**:

```sql
SELECT connection, name, ddc, i2c, COUNT((connection, ddc))
FROM (SELECT (CASE "kDisplayTransportType"
            WHEN 1 THEN 'DisplayPort'
            WHEN 3 THEN 'HDMI'
            ELSE 'Unknown'
        END) AS CONNECTION,
       name, ddc, i2c
FROM displays
WHERE "name" ILIKE 'HP U28%'
  AND "kCGDisplayID" <= 10 -- M1 assigns IDs from 2 to 10, Intel uses much larger IDs
  AND "DisplayProductID" != 0 -- Filter out those in a semi-connected state
) AS hp
GROUP BY (connection, name, ddc, i2c)
ORDER BY 5 DESC
```

<img width="1554" alt="image" src="https://user-images.githubusercontent.com/3819725/160296587-f775fbdb-9506-48c9-82db-4a6a41e6780d.png">

The data is pretty raw as you can see, it could be a lot more useful if we would combine it with some MonitorControl data and create some easy to understand dashboard.

**Reply by JoniVR** [reply]
Perhaps an open source effort could be launched by MonitorControl & Lunar to create an open source database with a simple frontend? 

There are other tools that provide dec control on windows and linux too so they might want to contribute too. 

If we set up some form of anonymous data collection this could really benefit users.
